package.json

```diff
@@ -151,14 +151,12 @@
     "exports-loader": "^3.0.0",
     "expose-loader": "^3.0.0",
     "grunt-cli": "^1.4.3",
-    "husky": "^6.0.0",
```

Comment:
This looks more like you removed husky instead of updating. I see only deletions even in package-lock.json. Is that expected?